### ISSUE TITLE
Issues with z-index when used with Drawer component Material UI

### ISSUE THREAD

**Issue by eryugal** [evandromacedo/react-simple-snackbar#2]
Snackbar is shown behind the Drawer and is visible on dismissing it, is there any way to set the z-z-index priority, I tried using, but it didn't help.
`const options = {
    zIndex: '10000 !important',
  }
....
    const [openSnackbar, closeSnackbar] = useSnackbar(options)
`
Any suggestions would be great!

**Comment by evandromacedo** [comment]
@yugalkishoregoyal yeah, sometimes these z-index issues are hard to deal, mainly when it involves another libraries.

Have you tried using the [Material UI Snackbar](https://material-ui.com/components/snackbars/) instead of the React Simple Snackbar? It could be better to integrate Material UI's own components instead of another libs in your project. Moreover, this component was created based on Material UI's snackbar, so they are very similar.

**Comment by ksdme** [comment]
@evandromacedo How about adding a dummy classname to https://github.com/evandromacedo/react-simple-snackbar/blob/master/src/Snackbar.js#L95 so global styling can set the z-index of the container? I realize that it is a hack but I also believe that snackbar z-index isn't something that we often style on a per component/usage basis.

Right now, having I have to use to get it to work, 
```css
div[class^="Snackbar_snackbar-wrapper"] {
  z-index: 1100;
}
```
This is not a very good solution as CSS minify or something other kind of transformations could break it.

**Comment by mattbbc** [comment]
> Right now, having I have to use to get it to work,
> 
> ```css
> div[class^="Snackbar_snackbar-wrapper"] {
>   z-index: 1100;
> }
> ```

This is just what I needed to get something working, thank you.

**Comment by scudette** [comment]
Thanks! This fixed my issue as well... A simple class name on the wrapper would solve the issue.
